### ISSUE TITLE
Branching issue when revisiting

### ISSUE THREAD

**Issue by swashbuck** [adaptlearning/adapt-contrib-bookmarking#69]
### Subject of the issue
When revisiting a course, blocks that have been skipped initially due to a correct answer are displaying.

Not sure if this is an issue with `adapt-contrib-branching` or `adapt-contrib-bookmarking`.

### Your environment
* Branching v1.2.2
* Bookmarking v4.2.1
* Framework 5.38.2

### Steps to reproduce
In _course.json_, enable branching:

```
  "_branching": {
    "_isEnabled": true
  }
```

Also, set up bookmarking:

```
  "_bookmarking": {
    "_isEnabled": true,
    "_level": "component",
    "_showPrompt": true,
    "title": "Resume?",
    "body": "Would you like to resume the course from the location you were at last time?",
    "_buttons": {
      "yes": "Yes",
      "no": "No"
    }
  },
```

And Spoor:

```
  "_spoor": {
    "_isEnabled": true,
    "_tracking": {
      "_shouldStoreResponses": true,
      "_shouldStoreAttempts": true,
      "_shouldRecordInteractions": true
    },
    "_reporting": {
      "_onTrackingCriteriaMet": "completed",
      "_onAssessmentFailure": "incomplete",
      "_resetStatusOnLanguageChange": false
    }
  },
```

In an **article**, enable branching:

```
    "_branching": {
      "_isEnabled": true,
      "_onChildren": true,
      "_start": ""
    },
```

In the **blocks** for this article, set up branching so that you jump to the last block on the page once you answer correctly. For example, below would be the `_branching` config for the first block `b-300`. If answered incorrectly, it would go to `b-305`. If answered correctly, it goes to `b-395`.

```
    "_branching": {
      "_containerId": "",
      "_correct": "b-395",
      "_partlyCorrect": "b-305",
      "_incorrect": "b-305"
    },
```
`b-305` would be configured similarly:

```
    "_branching": {
      "_containerId": "",
      "_correct": "b-395",
      "_partlyCorrect": "b-310",
      "_incorrect": "b-310"
    },
```

And so on. Set up a few blocks like this.

Finally, visit the course using the `scorm_test_harness.html` link or in an LMS. Correctly answer the first question in `b-300` so that you can skip the remainder of the blocks on the page (except for the last).

Revisit the page by reloading `scorm_test_harness.html` or exiting and returning to the LMS.

### Expected behaviour
Returning to the page should only show the first question (answered correctly) and the last block.

### Actual behaviour
The next question block `b-305` is displayed and is incomplete.

**Comment by oliverfoster** [comment]
Why is this posted in bookmarking? You've not included a reason.

**Comment by swashbuck** [comment]
> Why is this posted in bookmarking? You've not included a reason.

Yep, got lost in the weeds on this one. Bookmarking does _not_ affect the problem. I'll update the description and transfer to `adapt-contrib-branching`.
